src/CompileCommand.cpp

```diff
@@ -0,0 +1,242 @@
+#include "CompileCommand.hpp"
+
+#include "Error.hpp"
+#include "ccache.hpp"
+
+#include <third_party/llvm_yaml_escape.hpp>
+#include <third_party/nonstd/string_view.hpp>
+
+#include <map>
+
+// TODO "-fcolor-diagnostics" is added by ccache
```

Comment:
Need to check if `args_add(common_args, "-fcolor-diagnostics");` was called. If not, remove.